templates/kiali.yaml

```diff
@@ -0,0 +1,75 @@
+{{- if .Capabilities.APIVersions.Has "kiali.io/v1alpha1" }}
+apiVersion: kiali.io/v1alpha1
+kind: Kiali
+metadata:
+  name: kiali
+spec:
+  api:
+    namespaces:
+      exclude:
+      - "^kube-.+"
+  auth:
+    strategy: openid
+    openid:
+      client_id: "kiali"
+      disable_rbac: true
+      insecure_skip_verify_tls: true
```

Comment:
Why we need insecure here?

Reply:
good point we have to parameterize it for the local cluster. There we need insecure.